pkg/controllers/disruption/helpers.go

```diff
@@ -79,7 +79,8 @@ func SimulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 		pods = append(pods, n.reschedulablePods...)
 	}
 	pods = append(pods, deletingNodePods...)
-	scheduler, err := provisioner.NewScheduler(logging.WithLogger(ctx, operatorlogging.NopLogger), pods, stateNodes)
+	discardLog := logr.New(logr.Discard().GetSink())
```

Comment:
Same comment as the other one. If we initialize this in the `logging.go` package, we don't have to keep re-creating it every time that we run through this loop